executorlib/base/executor.py

```
@@ -178,3 +178,110 @@ def __del__(self):
+                                      gpus_per_worker: 0,
+                                      oversubscribe: False,
+                                      cwd: None,
+                                      executor: None,
+                                      hostname_localhost: False,
+                                  }
+
+        Returns:
+            Future: A Future representing the given call.
+        """
+        return self._executor.submit(
+            fn=fn, *args, resource_dict=resource_dict, **kwargs
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)